### PR TITLE
Chore: add snippets for the essentials docs section for 6.0

### PR DESCRIPTION
**Pull request by jonniebigodes**
With this pull request the snippets required for the essentials section are included and the markdown files are now adjusted to reflect the changes:

What was done:

- Updated the markdown files to include the snippets based on the file naming and structure decided.
- Created the snippets.

Feel free to provide feedback

### CODE REVIEW

**Review by domyen — APPROVED** [review]

File: docs/snippets/common/button-story-hide-nocontrols-warning.js.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+```js
+// Button.stories.js
+
+Basic.parameters = {
```
Comment (tmeasday) [inline docs/snippets/common/button-story-hide-nocontrols-warning.js.mdx:4]: I guess we should be consistent here -- in the other cases we include the `Template.bind({});` also.

File: docs/snippets/common/gizmo-story-controls-customization.js.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+```js
+// Gizmo.story.js
```
Comment (tmeasday) [inline docs/snippets/common/gizmo-story-controls-customization.js.mdx:2]: `.stories.js`?

File: docs/snippets/common/my-component-story-configure-viewports.js.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    viewport: {
+      // the viewports you want to use
+      viewports: MINIMAL_VIEWPORTS,
+      // your own default viewport
+      defaultViewport: 'iphone6'
+    },
+  };
+};
+
+export const myStory = () => <div />;
```
Comment (tmeasday) [inline docs/snippets/common/my-component-story-configure-viewports.js.mdx:17]: React code

File: docs/snippets/common/table-story-fully-customize-controls.js.mdx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+```js
+// Table.stories.js
+
+const TableStory = ({ data, ...args }) => (
+  <Table {...args} >
+    {data.map(row => (<TR>{row.map(item => <TD>{item}</TD>}</TR>))}
+  </Table>
+)
```
Comment (tmeasday) [inline docs/snippets/common/table-story-fully-customize-controls.js.mdx:8]: This is definitely a react snippet ;)

**Review by tmeasday — CHANGES_REQUESTED** [review]
Looks good, a couple changes.

File: docs/snippets/common/table-story-fully-customize-controls.js.mdx (hunk 4 above)
Comment (jonniebigodes) [reply docs/snippets/common/table-story-fully-customize-controls.js.mdx]: Fixed with the latest commit

File: docs/snippets/common/my-component-story-configure-viewports.js.mdx (hunk 3 above)
Comment (jonniebigodes) [reply docs/snippets/common/my-component-story-configure-viewports.js.mdx]: Fixed with the latest commit

File: docs/snippets/common/gizmo-story-controls-customization.js.mdx (hunk 2 above)
Comment (jonniebigodes) [reply docs/snippets/common/gizmo-story-controls-customization.js.mdx]: Fixed with the latest commit

File: docs/snippets/common/button-story-hide-nocontrols-warning.js.mdx (hunk 1 above)
Comment (jonniebigodes) [reply docs/snippets/common/button-story-hide-nocontrols-warning.js.mdx:6]: Fixed with the latest commit

**Pull request merged**